src/styles/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  margin-right: 4px;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
Move these styles to stars_star class, there is no need to create separate BEM block for one star